app/views/govuk_publishing_components/components/docs/image_card.yml

```
@@ -283,3 +293,7 @@ examples:
         /assets/govuk_publishing_components/image-card-srcset/cop26-320.jpg: 320w
         /assets/govuk_publishing_components/image-card-srcset/cop26-240.jpg: 240w
         /assets/govuk_publishing_components/image-card-srcset/cop26-170.jpg: 170w
+    embed: |
+      <div class="govuk-!-width-full">
```

I think we should wrap this in a one-third container, otherwise it looks odd when it goes full width:

```suggestion
      <div class="govuk-!-width-one-third">
```

---

Actually, ignore me on this! The image diff shows the main branch version at full width - the image is broken on the live site which confused me.